src/systemcmds/gpio/Kconfig

```diff
@@ -2,4 +2,11 @@ menuconfig SYSTEMCMDS_GPIO
 	bool "gpio"
 	default n
 	---help---
-		Enable support for gpio
\ No newline at end of file
+		Enable support for gpio
+
+menuconfig USER_GPIO
```

Comment:
This one accesses gpios directly.

Reply:
> I suppose this makes sense, it just seems kind of noisy for something I assumed wasn't really going to change.
> 
> Are you expecting it to vary between builds or does it end up being pretty obvious which modules need to run kernel side? I guess I'm wondering if we could simply hard code the most likely default at some level (px4_add_module) and then only supply the kconfig option for the few cases where it might change?

I wouldn't expect it to change between builds, and it should be pretty obvious which modules need to run in kernel and which can be selected. 

Thanks for pointing out the obvious mistakes! I added the option initially (via scripting) for all "modules" and "systemcmds" and then removed the option from the ones which obviously can't go to user side. I just missed the ones!

The default is "kernel", if this selection doesn't exist at all